File: CTDK_Python/139/_139.py
from math import sqrt

a = float(input("Nhap a: "))
b = float(input("Nhap b: "))

if (a == 0):
    if (b == 0):
        print("Vo so nghiem!!!")
    else:
        print("Vo nghiem!!!")
else:
    x = -b / a
    print("Nghiem: ", x)

File: CTDK/139/139.cpp
#include <iostream>
using namespace std;

int main()
{
	float a;
	cout << "Nhap a: ";
	cin >> a;

	float b;
	cout << "Nhap b: ";
	cin >> b;

	float f;
	if (a == 0)
		if (b == 0)
			cout << "Vo so nghiem" << endl;
		else
			cout << "Vo nghiem" << endl;
	else
	{
		float x = -b / a;
		cout << x << endl;
	}

	return 0;
}